### ISSUE TITLE
Passing extensions in via command line

### ISSUE THREAD

**Issue by thisispaul** [JohnPostlethwait/stringify#28]
Sorry if this is the wrong place to put this but having trouble trying to figure out what is going on (it may well be I'm making a mistake)

I'm trying to pass extensions in via the command line like so:

`browserify -t [ stringify --extensions ['.hjs', '.html', '.whatever'] ]`

But when I do this i get this error:

```
79: extensions = extensions.map(function (ext) {
                          ^
TypeError: undefined is not a function
```

It is entirely plausible that I am passing in these options incorrectly but I can't find any documentation on the right way to pass them in via command line. (I have also tried the extensions without the square brackets, without commas and without inverted commas)

**Comment by JohnPostlethwait** [comment]
Hello! Yes, this is the correct place to ask.

Stringify (unfortunately) does not currently accept arguments from the command-line. :( When run from the command-line it just uses its defaults. for extensions (and all other options). Command-line support was sort of a happenstance and I didn't put much work into it.

I think it is a good idea to have it. It would be great if you wanted to add a pull request for it, otherwise I can work on it eventually, but it likely won't be any time soon...

**Comment by thisispaul** [comment]
Okay, that's fine, mostly just glad I wasn't doing something daft!

 I'm not 100% on how to add it myself but I shall investigate and give it a shot!

**Comment by JohnPostlethwait** [comment]
Haha, nope, nothing daft!

You could add something here for looking at environmental variables, if you wanted to do it that way: https://github.com/JohnPostlethwait/stringify/blob/master/src/stringify.js#L63 Supporting actual command-line args would be a bit more involved.

**Comment by Samsy** [comment]
https://github.com/JohnPostlethwait/stringify/pull/29

**Comment by Samsy** [comment]
https://github.com/JohnPostlethwait/stringify/pull/30

**Comment by Samsy** [comment]
Glad to merge my P-Request, you can close this one now ! 
Cheers !

**Comment by Samsy** [comment]
Can you publish to npm plz.
Thanks

**Comment by JohnPostlethwait** [comment]
Sorry, have been quite distracted and busy. I am working on getting this updated right now. I will let you know what it is up.

**Comment by JohnPostlethwait** [comment]
Published as version 3.2.1 https://www.npmjs.com/package/stringify

I will be pushing a version 4 later that has a newer version of NodeJS
